Portfolio/index.html

```diff
@@ -6,8 +6,98 @@
     <link rel="stylesheet" href="index.css" />
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <!-- Write your code below for PORTFOLIO -->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+  <body style="margin: 0; padding: 0px">
+    <h2>Juneja.</h2>
+<ul>
```

Comment:
Consider adding more descriptive text for the button, such as 'Contact Me' instead of 'Contact'